java/restheart/pom.xml

```diff
@@ -7,8 +7,7 @@
   <packaging>jar</packaging>
 
   <properties>
-    <maven.compiler.source>21</maven.compiler.source>
-    <maven.compiler.target>21</maven.compiler.target>
+    <maven.compiler.release>21</maven.compiler.release>
```

Comment:
You can create a parent POM that defines common options (see [introduction to the POM](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html)). 

However, since different frameworks have very distinct dependencies and options, I suspect it may not be very useful.